Dockerfile.run

```diff
@@ -8,6 +9,7 @@ ARG pack_gid=1000
 RUN groupadd heroku --gid ${pack_gid} && \
   useradd heroku -u ${pack_uid} -g ${pack_gid} -s /bin/bash -m
 
-LABEL io.buildpacks.stack.id="heroku-18"
+ARG STACK
+LABEL io.buildpacks.stack.id="${STACK}"
```

Comment:
I don't want to block @edmorley on this, but switching to `heroku/heroku:20` might be a good opportunity for pack-images and our CNBs to move away from `stack = "heroku-20"` and towards `stack = "io.buildpacks.stacks.bionic"` plus mixins.

Reply:
Example:
https://github.com/paketo-buildpacks/stacks/blob/main/bionic/cnb/run/Dockerfile#L13-L14

Reply:
@jkutner Should I ask the CNB maintainers to hold off adding `stack = "heroku-20"` then? I've just filed a bunch of work items asking them to add it, and https://github.com/heroku/procfile-cnb/pull/5 has already merged.

Or is such a move something we can do in a non-breaking way for both heroku-18 and heroku-20 in the future? Since `builder.toml` contains versioned CNB URLs, it seems we can switch over the builder to use the new IDs in one go in the future, without breakage?

Reply:
We should have the buildpacks add `heroku-20`, so those work items are 👌 We can migrate without breaking anything.

(Sidenote: Actually, we really want to move `io.buildpacks.stacks.focal`)

Reply:
If the referenced PRs aren't merged, we might consider adding `io.buildpacks.stacks.focal` and `io.buildpacks.stacks.bionic` as supported stacks in those buildpack PRs. It may save some time in the future and unless someone is building their own builder, it shouldn't have any consequences.

Reply:
@jabrown85 but that would be a lie in most cases, because they also require some mixins. maybe better to wait until the buildpack owners can add both?

Reply:
ah, yeah. That is the part that was _super_ annoying when we tried to first make the switch to bionic. It's whack-a-mole trying to figure out which mixins you require to be present.

Reply:
Yeah I think since we don't yet have the list of mixins, it would be best to hold off on buildpacks declaring support for `io.buildpacks.stacks.focal`.

Since the conversation about changing stack name/label has moved to email, and that we don't want to block on it, are we good to call this PR thread resolved, and to continue with this PR? :-)

Reply:
Yes, we should move forward with this PR